### ISSUE TITLE
keyboard input disabled for window

### ISSUE THREAD

**Issue by aw-bib** [vimperator/vimperator-labs#565]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:47.0) Gecko/20100101 Firefox/47.0
```

##### Description:

Usually, I use two Firefox windows: 
- normal window
- private window
on separate workspaces in xfce4. All of those hold several tabs. The first one even one tab that ignores vimperator completely. (Zimbras keybindings collide with vimperator.)

After some time it happens that one of the windows does not accept any keyboard input. I can use the mouse but the keyboard is lost and does not come back. Usually, I can close the affected window, switch to the other one which has still keyboard input and reopen it. It only happens on the basis of separate windows, I did never see it to happen on a tab only.

I did not see this behaviour in the past (using vimperator for about a year now) so it seems to be some recent upgrade either to Firefox or vimperator.

##### Expected behavior:

Do not loose keyboard input in windows ;)

##### Steps to reproduce:

It happens quite often but till now I can not find a real trigger (website etc.)

**Comment by timss** [comment]
Sounds similar to previous issues which are possibly related to window management, although it doesn't have to be this exclusively.

For now, please see #39. 

I've had this happen from time to time on different setups as well, but it's usually™ solved by toggling focus such as `<A-t>` (<kbd>L-Alt</kbd>).

**Comment by aw-bib** [comment]
Agree, #39  also #317 sounds indeed similar. Seems to be a dupe of them. Unfortunatley, I did not find them in the tracker before you pointed me to #39 . I'll see if some of the mentioned work arounds "solve" it. As far as I can see till now, viewing page info as mentioned in #39 looks promising, at least it unstuck it successfully. Interestingly enough I never had any issues like that till quite recently.

**Comment by timss** [comment]
>Interestingly enough I never had any issues like that till quite recently.

Sounds familiar indeed, many have reported that it "comes and goes" and other than it possibly has something to do with window management I've not been able to track down any common factors between the issues that could point out a place to start debugging.

**Comment by aw-bib** [comment]
Indeed, it is quite nasty and persistent. 

Additionally, it seems, that opening the page info re-enables normal input behaviour. This seems not to the case while activating a menue. Alt-t does not work as this keystroke is not recognized. Sometimes I can hit Alt and then t would open the tools menu. The menu bar would not appear at however at Alt (I've hidden it), and I can release Alt before hitting t, ie similar to caps.)

What is indeed quite interesting is that it comes in some cases with a focus issue as well. Sometimes it seems that the FF menu bar has the focus or some Alt/Ctrl-Lock. Sometimes it seems that "the other FF window" has the focus. Ie. input in the locked window triggers events in the remote, still functioning one.

Don't know if this helps.
